src/includes/classes/AutoCache.php

```diff
@@ -282,6 +285,11 @@ protected function getSitemapUrlsDeep($sitemap, $___recursive = false)
             }
             unset($_sitemapindex_urls, $_urlset_urls);
         }
+
+        if (!$allow_url_fopen) {
```

Comment:
Does this conditional check that deletes the temporary file also consider the case of the filter being used?